.github/actions/setup_ubuntu/action.yml

```diff
@@ -2,12 +2,13 @@ name: setup_ubuntu
 inputs:
   cwd:
     required: false
-    default: "${PWD}"
+    default: ${{ github.workspace }}
```

Comment:
See above. I needed to change this to `${{ github.workspace }}` for ubuntu but this doesn't work on windows so windows builds kept `${PWD}`

Reply:
Update. This seems to be the one place where `${PWD}` doesn't work. For all actions running on Windows, ${{ github.workspace }} does not work (because unix style filepath) but `${PWD}` works. For other actions such as linting or unit_tests, both `${PWD}` and ${{ github.workspace }} work on ubuntu.

Reply:
Would the [`GITHUB_WORKSPACE` environment variable](https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables) be correct on windows?

Reply:
I think GITHUB_WORKSPACE and github.workspace resolve to the same thing but I might be misunderstanding the doc.

Reply:
@vcua-mobify There is the [`create-mrt` action](https://github.com/SalesforceCommerceCloud/pwa-kit/actions/runs/6739547104/job/18325542066) that failed because of `PWD`. The error message:
`Error: An error occurred trying to start process '/usr/bin/bash' with working directory '/home/runner/work/pwa-kit/pwa-kit/${PWD}'. No such file or directory`

This error is preventing the later actions (in particular, publish to npm) that is necessary for us at this moment.